graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -624,11 +665,32 @@ def _compile_root_ast_to_ir(schema, ast, type_equivalence_hints=None):
     # Construct the start location of the query, and the starting context object.
     location = Location((base_start_type,))
     context = {
+        # 'tags' maps to a dict containing
+        #  - location: Location where the tag was defined
+        #  - optional: boolean representing whether the tag was defined within an @optional scope
+        #  - type: GraphQLType of the tagged value
         'tags': dict(),
+        # 'outputs' maps to a dict mapping each output name to another dict which contains
```

Comment:
I would generally avoid saying `maps to` where you mean `is`, since we're already talking about dicts mapping keys to values, and we want to minimize confusion.